fabric-game-rule-api-v1/src/main/resources/fabric-game-rule-api-v1.accesswidener

```diff
@@ -0,0 +1,5 @@
+accessWidener	v1	named
+accessible  method  net/minecraft/world/GameRules$Type <init>  (Ljava/util/function/Supplier;Ljava/util/function/Function;Ljava/util/function/BiConsumer;Lnet/minecraft/world/GameRules$Acceptor;)V
+extendable  method  net/minecraft/world/GameRules$Type <init>  (Ljava/util/function/Supplier;Ljava/util/function/Function;Ljava/util/function/BiConsumer;Lnet/minecraft/world/GameRules$Acceptor;)V
+accessible  class   net/minecraft/world/GameRules$Acceptor
+accessible  class   net/minecraft/client/gui/screen/world/EditGameRulesScreen$RuleWidgetFactory
```

Comment:
irritatered, mixin good aw bad, some of these are unnecessary anyways, long live @Coerce

Reply:
nope, private ctors in GameRules$Type

Reply:
accessible  class   net/minecraft/world/GameRules$Acceptor
accessible  class   net/minecraft/client/gui/screen/world/EditGameRulesScreen$RuleWidgetFactory
also invokers exist

Reply:
have you heard of our lord and savior "type = "path/to/private/class"", and @Coerce

Reply:
package-private inner class and inner class package-private. I have to implement RuleWidgetFactory and acceptor for widgets and custom rules